packages/legend-studio/src/components/editor/edit-panel/GrammarTextEditor.tsx

```diff
@@ -199,7 +199,7 @@ export const GrammarTextEditor = observer(() => {
       if (error?.sourceInformation) {
         setErrorMarkers(
           editorModel,
-          error.message,
+          error.message === '' ? error.nonEmptyMessage : error.message,
```

Comment:
I think we don't need this, let's just hard code the error here `(no error message)`

Reply:
will update it.